cmd/scheduler.go

```diff
@@ -82,4 +85,11 @@ func init() {
 		"api key to orchard service",
 	)
 	viper.BindPFlag("scheduler.orchard.apiKey", schedulerCmd.Flags().Lookup("orchard"))
+
+	schedulerCmd.Flags().Duration(
+		"lockTTL",
+		time.Hour,
```

Comment:
is one hour a good default duration for the LockTTL?

Reply:
I think more flexible to have in units of time.Minute

Reply:
Agreed, done.